pkg/versionstream/versionstreamrepo/gitrepo_integration_test.go

```diff
@@ -0,0 +1,70 @@
+// +build integration
+
+package versionstreamrepo_test
+
+import (
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/gits"
+	"github.com/jenkins-x/jx/pkg/versionstream/versionstreamrepo"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	RepoURL    = "https://github.com/jenkins-x/jenkins-x-versions"
+	VersionRef = "v1.0.90"
+	BranchRef  = "master"
+	HEAD       = "HEAD"
+)
+
+func TestCloneJXVersionsRepoWithATag(t *testing.T) {
```

Comment:
We should also test:

* Different URL than the default
* A URL and ref set in the team settings
* Not passing in a URL at all (to validate the default)

Reply:
Got it, will do.